### PR TITLE
Add lookups view to allow adding/editing/deleting of druid lookups

### PR DESCRIPTION
**Pull request by shuqi7**
Refers to Druid lookups: http://druid.io/docs/latest/querying/lookups.html

- Lookups page to allow users to view existing lookups, add new lookups, and edit lookups
image
- Allow users to use ISO string as current version, and will pre-fill current ISO as version when users add new lookups
image
- Spec refers to the JSON object in `lookupExtractorFactory`
- If the user has NEVER configured lookups before, he must initialize the configuration first
image

### CODE REVIEW

**Comment by vogievetsky** [comment]
I have reviewed these changes already and it has my 👍
Some of my review comments (that have already been addresses) can be found here: https://github.com/implydata/druid/pull/19

File: web-console/src/dialogs/lookup-edit-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          onChange={(e: any) => onChange("lookupEditVersion", e.target.value)}
+          placeholder={"Enter the lookup version"}
+          rightElement={<Button className={"pt-minimal"} text={"Use ISO as version"} onClick={() => this.addISOVersion()} />}
+        />
+      </FormGroup>
+
+      <FormGroup className={"lookup-label"} label={"Spec:"}/>
+
+      <AceEditor
+        className={"bp3-fill lookup-edit-dialog-textarea"}
```
[the remark this answers is not in the document]
Comment (shuqi7) [reply web-console/src/dialogs/lookup-edit-dialog.tsx]: Should be. Must have missed this after downgrading from bp3.

**Comment by clintropolis** [comment]
Is there a reason why the dialog lookup editor doesn't take up more screen width? Might make it a bit nicer to work with.

Screen Shot 2019-03-15 at 4 54 33 PM

**Comment by shuqi7** [comment]
> Is there a reason why the dialog lookup editor doesn't take up more screen width? Might make it a bit nicer to work with.
> 
> Screen Shot 2019-03-15 at 4 54 33 PM

Made it wider by setting it to 600px, which I think is more user-friendly while still looking pretty

**Pull request merged**